folly/io/async/SSLContext.h

```diff
@@ -574,6 +574,15 @@ class SSLContext {
 
   [[deprecated("Use folly::ssl::init")]] static void initializeOpenSSL();
 
+#if FOLLY_OPENSSL_HAS_TLS13
+  /**
+   * Callback to parse 0-RTT early data.
+   */
+  std::function<bool(
+      const unsigned char *buf,
+      size_t bufSize)> parseEarlyDataCb_;
```

Comment:
This can probably be merged into `AsyncSSLSocket::HandshakeCB` (maybe something like `handshakeEarlyData(folly::Range)`)?